charts/logging/loki/values.yaml

```diff
@@ -80,23 +83,26 @@ loki:
       working_directory: /data/loki/boltdb-shipper-compactor
       shared_store: filesystem
 
-  image:
-    repository: docker.io/grafana/loki
-    tag: 2.1.0
-    pullPolicy: IfNotPresent
-
   ## The app name of loki clients
   client: {}
     # name:
 
   nodeSelector: {}
 
   persistence:
+    enabled: true
     size: 15Gi
     storageClassName: kubermatic-fast
 
   podLabels: {}
 
+  podAnnotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "3100"
+
+  rbac:
+    pspEnabled: false
+
```

Comment:
I think we should move these into a separate PR, as they change application behaviour and go beyond using the upstream helm chart. If that is too much of a hassle with changing all the test outputs, then I would at least place them in a separate commit. Even though we squash all commits, I think it still makes it easier to understand what the PR has changed.

Reply:
I'm pretty sure it just puts everything to the way it was before switching to the upstream, e.g. to the "defaults" that we were using, [output diff](https://github.com/kubermatic/kubermatic/pull/8252/commits/254140d41667c03809ee1af334d9706a2b6e4c64#diff-eca8a05f6cbb885406e88aee2f5b8acbda5b4bd4f295790774a3157ef2049cd5) seems to confirm that unless I'm missing something?

Reply:
Okay maybe I misunderstood then, but my thinking was the following. Let's take the `persistence.enabled` as an example:
- I looked through our [current values](https://github.com/kubermatic/kubermatic/blob/master/charts/logging/loki/values.yaml), where it's not set to anything
- Then I looked up the [old upstream](https://github.com/grafana/helm-charts/blob/loki-2.3.0/charts/loki/values.yaml#L139) (which I assumed is the one we cloned) where it is set to false
- additionally I looked through the values of the new [upstream chart](https://github.com/grafana/helm-charts/blob/main/charts/loki/values.yaml#L142), where it is also set to `false`

And then I came to the conclusion that this would be set to the opposite. But maybe I overlooked something and we are setting the value to `true` somewhere in our old chart 🤔 ?

Reply:
Our currently used chart is not directly copied from the upstream, so there are some differences based on assumptions in KKP. Our current chart does not need need to have `persistence.enabled` because it is enough to define a PVC to use it. In the upstream chart, you have to add `persistence.enabled` AND define a PVC for it to work. Our charts are basically a set of default configurations and optional additions to upstream charts, so to keep compatibility with older charts, some options have to be set differently than in the upstream.

The link to diff in test output is the best way to see what has been changed, as it shows the effect of installing the chart with default values.